cpp/example_code/s3/get_put_bucket_acl.cpp

```diff
@@ -272,15 +272,21 @@ int main()
     Aws::SDKOptions options;
     Aws::InitAPI(options);
     {
-        const Aws::String bucket_name = "my-bucket";
-
-        // Set the ACL's owner information. 
+        //TODO: Change to the name of an actual bucket in your account.
```

Comment:
Use consistent descriptions. For example:
// Change the value of bucket_name to the name of a bucket in your account.

I'm not sure we need to tell them that if they supply an incorrect name they'll get an error. Seems obvious to me.